### PR TITLE
add a new weekly pipeline to update the glass binaries from main bran…

### PR DESCRIPTION
**Pull request by bschnurr**
…ch. (#8104)

user PRs weren't able to auth

### CODE REVIEW

**Review by AdamYoblick — APPROVED** [review]

File: Build/glass/cache_glass.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  # Setup the glass test folder
+  - task: PythonScript@0
+    displayName: 'Setup glass test folder'
+    env:
+      SYSTEM_ACCESSTOKEN: $(DropToken)
+    inputs:
+      scriptSource: 'filePath' # 'filePath' | 'inline'. Required. Script source. Default: filePath.
+      scriptPath: $(Build.SourcesDirectory)/Build/setup_glass.py
+      arguments: '--authTokenVariable SYSTEM_ACCESSTOKEN --buildOutput $(Build.BinariesDirectory)\raw\binaries'
```
Comment (rchiodo) [inline Build/glass/cache_glass.yml:32]: I'm confused, did you submit this somewhere else?

Anyway, we might want to modify setup-glass.py to take another argument that skips copying the PTVS build output.

Comment (bschnurr) [reply Build/glass/cache_glass.yml:32]: i submitted a PR to this branch that is in ptvs instead of my own.

Comment (bschnurr) [reply Build/glass/cache_glass.yml:32]: but i forgot that the cache pipeline wont run until i check it in.

**Review by bschnurr — COMMENTED** [review]
disable glass test for now

**Review by rchiodo — APPROVED** [review]

**Pull request merged**